apps/avatax/src/pages/api/webhooks/checkout-calculate-taxes.ts

```diff
@@ -10,86 +10,95 @@ import { ObservabilityAttributes } from "@saleor/apps-otel/src/lib/observability
 import { loggerContext } from "../../../logger-context";
 import { checkoutCalculateTaxesSyncWebhook } from "../../../modules/webhooks/definitions/checkout-calculate-taxes";
 import { verifyCalculateTaxesPayload } from "../../../modules/webhooks/validate-webhook-payload";
+import { metadataCache, wrapWithMetadataCache } from "../../../lib/app-metadata-cache";
 
 export const config = {
   api: {
     bodyParser: false,
   },
 };
 
+const withMetadataCache = wrapWithMetadataCache(metadataCache);
+
 /**
  * TODO: Add tests to handler
  */
 export default wrapWithLoggerContext(
   withOtel(
-    checkoutCalculateTaxesSyncWebhook.createHandler(async (req, res, ctx) => {
-      const webhookResponse = new WebhookResponse(res);
-
-      try {
-        const logger = createLogger("checkoutCalculateTaxesSyncWebhook");
-        const { payload } = ctx;
-
-        loggerContext.set("channelSlug", ctx.payload.taxBase.channel.slug);
-        loggerContext.set("checkoutId", ctx.payload.taxBase.sourceObject.id);
-        if (payload.version) {
-          Sentry.setTag(ObservabilityAttributes.SALEOR_VERSION, payload.version);
-          loggerContext.set(ObservabilityAttributes.SALEOR_VERSION, payload.version);
-        }
-
-        logger.info("Handler for CHECKOUT_CALCULATE_TAXES webhook called");
+    withMetadataCache(
```

Comment:
only change for handlers is adding this wrapper to handler. rest of the diff is whitespace